it/test/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/controllers/AppealStartControllerISpec.scala

```diff
@@ -19,9 +19,10 @@ package uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers
 import org.jsoup.Jsoup
 import play.api.http.Status.OK
 import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.stubs.AuthStub
-import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.utils.{ComponentSpecHelper, ViewSpecHelper}
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.utils.{ComponentSpecHelper, ViewSpecHelper, NavBarTesterHelper}
 
-class AppealStartControllerISpec extends ComponentSpecHelper with ViewSpecHelper with AuthStub {
+
+class AppealStartControllerISpec extends ComponentSpecHelper with ViewSpecHelper with AuthStub with NavBarTesterHelper {
 
 
   "GET /appeal-start" should {
```

Comment:
Need to add call to test the navBar, with the following method from the NavBarTesterHelper:

```suggestion
  "GET /appeal-start" should {
  
     testNavBar("/appeal-start")()
```